client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -102,10 +109,12 @@ class PurchasesListing extends Component {
 			return getJetpackProductTagline( purchase );
 		}
 
+		const jetpackBackup = this.getJetpackBackup();
```

Comment:
Same comment regarding naming applies here - should `jetpackBackup` be `jetpackBackupPurchase`?